CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contribution Guide
+
+- [Creating a Pull Request](#creating-a-pull-request)
+- [Local Development Setup With Docker Compose](#local-development-setup-with-docker-compose)
+- [Local Development Setup Without Docker Compose](#local-development-setup-with-docker-compose)
+- [Running Tests](#running-tests)
+- [Linting with Rubocop](#linting-with-rubocop)
+- [Setup Required ENVs](#setup-required-envs)
+
+## Creating a Pull Request
+
+Before you open a pull request, please ensure that the test suite passes and there are no rubocop infractions, as these will be run by GitHub Actions before merging.
+
+Overview:
+-   To work on something new, create a descriptively named branch off of main (ie: add-new-feature)
+-   Commit to the new branch locally and regularly push your work to the same named branch on GitHub
+-   If you need feedback or help, or you think a branch is ready to merge, open a pull request
+-   Remediate issues uncovered by GitHub Actions linters & automated testing
+-   Wait for code review & suggestions
+
+## Local Development Setup with Docker Compose
+
+1. [Install Docker Desktop](https://www.docker.com/products/docker-desktop/)
+1. Build Docker Images Locally: `docker compose build`
```

Comment:
While attempting `docker compose build` on M1 MacBook Pro, docker build fails with error on step 3/7:
```
ERROR [bonne-bouche-app 3/7] RUN curl -sL https://dl.google.com/linux/linux_signing_key.pub > /etc/apt/trusted.gpg.d/google_linux_signing_key.asc     && echo "deb http://dl.google.com/linux/chrome/deb/ stable main" | tee /etc/apt/sources.list.d/google_chrome.list     && apt-get update -qq     && ap  2.6s
 => [bonne-bouche-css internal] load build context                                                                                                                                                                                                                                                               0.0s
 => => transferring context: 4.44kB                                                                                                                                                                                                                                                                              0.0s
 => [bonne-bouche-js internal] load build context                                                                                                                                                                                                                                                                0.0s
 => => transferring context: 4.44kB                                                                                                                                                                                                                                                                              0.0s
 => [bonne-bouche-web internal] load build context                                                                                                                                                                                                                                                               0.0s
 => => transferring context: 4.44kB                                                                                                                                                                                                                                                                              0.0s
------
 > [bonne-bouche-app 3/7] RUN curl -sL https://dl.google.com/linux/linux_signing_key.pub > /etc/apt/trusted.gpg.d/google_linux_signing_key.asc     && echo "deb http://dl.google.com/linux/chrome/deb/ stable main" | tee /etc/apt/sources.list.d/google_chrome.list     && apt-get update -qq     && apt-get install -y     google-chrome-stable     && apt-get clean     && rm -rf /var/lib/apt/lists/* /usr/share/doc /usr/share/man:
#0 0.308 deb http://dl.google.com/linux/chrome/deb/ stable main
#0 2.037 Reading package lists...
#0 2.323 Building dependency tree...
#0 2.404 Reading state information...
#0 2.458 E: Unable to locate package google-chrome-stable
------
failed to solve: executor failed running [/bin/sh -c curl -sL https://dl.google.com/linux/linux_signing_key.pub > /etc/apt/trusted.gpg.d/google_linux_signing_key.asc     && echo "deb http://dl.google.com/linux/chrome/deb/ stable main" | tee /etc/apt/sources.list.d/google_chrome.list     && apt-get update -qq     && apt-get install -y     google-chrome-stable     && apt-get clean     && rm -rf /var/lib/apt/lists/* /usr/share/doc /usr/share/man]: exit code: 100
```

Reply:
Instructions are in the Required ENVs, will add to Docker Compose Setup as well